packages/mobile/src/send/utils.ts

```diff
@@ -210,15 +210,28 @@ export function* handleSendPaymentData(
       Logger.warn(TAG, '@handleSendPaymentData null amount')
       return
     }
-    const transactionData: TransactionDataInput = {
-      recipient,
-      amount,
-      reason: data.comment,
-      type: TokenTransactionType.PayPrefill,
+    if (data.asset === 'CELO') {
+      navigate(Screens.WithdrawCeloReviewScreen, {
+        amount,
+        recipientAddress: data.address.toLowerCase(),
+        feeEstimate: new BigNumber(0),
+      })
+    } else if (data.asset === 'cUSD' || !data.asset) {
+      const transactionData: TransactionDataInput = {
+        recipient,
+        amount,
+        reason: data.comment,
+        type: TokenTransactionType.PayPrefill,
+      }
+      navigate(Screens.SendConfirmation, { transactionData, isFromScan: true })
```

Comment:
Depending on what restriction we add on the schema for the `asset` field, we would need to handle the case where the asset is something unsupported (e.g. `BTC` 😛).
We could display our error banner `Unsupported assets XXX` for instance.

Reply:
Gotcha! Right now, if the deep links don't work, we don't provide any error message to the user -- the app simply doesnt change screens. Should I leave it for consistency, or would you like me to add error banners for the other cases too? (for example `celo://wallet/pay?amount=.1` is invalid as well, and right now nothing is shown on the user's end).

Reply:
I see yes, let's leave it at that for now, but could you open an issue to track this? 🙏 
Can be confusing for end users to expect something and nothing happens.

Reply:
Agree! Made #5688